docs/wcf_support.md

```diff
@@ -0,0 +1,41 @@
+
+> Note: At the time this documentation was written, the WCF updater extension is supplied as a Preview release. The list below summarizes the functionality and requirements for the Preview release version. When new features are added to the extension in the future, we will also update the list.
+## Supported functionality and minimum requirements
+
+- The latest version of .NET Upgrade Assistant with CoreWCF extension (Preview) from the [Azure Devops Pipeline](https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-tools)
+  - ✅ Upgrade csproj to the newer SDK style format
+  - ✅ Update project TFM to net6.0
+  - ✅ Update WCF project with a single `ServiceHost` instance and replace it with ASP .NET Core hosting.
+  - ✅ Update WCF project with multiple services and all `ServiceHost` are instantiated and configured in the same method.
+  - ✅ Update the original configuration file in the project and generate a new configuration file for CoreWCF.
+  - ✅ Replace `System.ServiceModel` namespace and references with CoreWCF ones in .cs and project files.
+  - ❌ WCF server that are Web-hosted and use .svc file.
+  - ❌ Behavior configuration except `serviceDebug`, `serviceMetadata`, `serviceCredentials`(`clientCertificate`, `serviceCertificate`, `userNameAuthentication`, and `windowsAuthentication`)
+  - ❌ Endpoints using bindings other than NetTcpBinding and HTTP based bindings
+
+
+- For the WCF project to be applicable for this upgrade, it must meet the following requirements:
+  - Includes a .cs file that references `System.ServiceModel` and creates new `ServiceHost`.
+    - If the WCF project has multiple `ServiceHost`, all hosts need to be created in the same method.
+  - Includes a .config file that stores `System.ServiceModel` properties
+
+> Note: If your project is not applicable for this tool, we recommend you to check out the [CoreWCF walkthrough guide](https://github.com/CoreWCF/CoreWCF/blob/main/Documentation/Walkthrough.md) and
+[BeanTrader Sample demo](https://devblogs.microsoft.com/dotnet/upgrading-a-wcf-service-to-dotnet-6/) and manually update the project.
+## Migration Resources
+
+To assist your migration process, please check out the following documentation and blogs:
+
+- Step-by-Step demo: [Upgrade WCF Server-side Project to use CoreWCF on .NET 6](Need to insert link here after that PR merged in)
+- Manual upgrade guide: [Upgrading a WCF service to .NET 6 with CoreWCF](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/migrate-to-windows-app-sdk-ovw)
+- CoreWCF Blog: [CoreWCF 1.0 has been Released, WCF for .NET Core and .NET 5+](https://devblogs.microsoft.com/dotnet/corewcf-v1-released/)
+
+After upgrading the project, you'll need to compile and test them. The upgrade assistant will do what it can, but it can't solve every incompatibility as part of the project upgrade. Please pay close attention to the upgrade-assistant console output and complete any manual updates requested by the tool.
```

Comment:
```suggestion
After upgrading the project, you'll need to compile and test them. Upgrade Assistant will do what it can, but it can't solve every incompatibility as part of the project upgrade. Please pay close attention to the upgrade-assistant console output and complete any manual updates requested by the tool.
```